libs/island-ui/core/src/lib/InputFileUpload/InputFileUpload.tsx

```diff
@@ -28,7 +29,7 @@ export const fileToObject = (
   status?: UploadFileStatus,
 ): UploadFile => {
   return {
-    name: file.name,
+    name: kebabCase(file.name),
```

Comment:
Let's store the filename as is (shown to the user) just modify the key we store it in s3 (server side, when creating the presigned post fields).